templates/primary-secondary/primary-secondary.js

```diff
@@ -74,6 +74,15 @@ class TemplatePrimarySecondary extends LitElement {
 					"secondary";
 					grid-template-columns: auto;
 					grid-template-rows: auto 1px auto;
+					overflow-y: auto;
+				}
+				main {
+					grid-area: primary;
+					overflow-y: visible;
+				}
+				aside {
+					grid-area: secondary;
+					overflow-y: visible;
```

Comment:
Do you need to specify the `grid-area` here? `main` should already be in `primary` and `aside` should already be in `secondary` from the CSS defined above.

Reply:
You're right - will put up a new commit without the grid-area specified again in the media query.